weatherman/utils/dateUtils.js

```diff
@@ -0,0 +1,38 @@
+import moment from "moment";
+
+/**
+ * @param { string } inputDate  - input date
+ * @param { string } inputFormat - format of input date
+ * @param { string } monthFormat - format of return month "MMMMM" in case of
+ * full month name "MMM" in case of short name
+ */
+export const parseMonthArgsInputToMonth = (
+  inputDate,
+  inputFormat = "Y-M-D",
+  monthFormat = "MMMM"
+) => {
+  const parsedMonthDate = moment(inputDate, inputFormat, true);
+  return parsedMonthDate.isValid()
```

Comment:
is ternary operator required here? Or something else can be more appropriate here?